inject/android.js

```diff
@@ -2,19 +2,10 @@
  * @module Android integration
  * @category Content Scope Scripts Integrations
  */
-import { load, init, update } from '../src/content-scope-features.js'
+import { load, init } from '../src/content-scope-features.js'
 import { processConfig, isGloballyDisabled } from './../src/utils'
 import { isTrackerOrigin } from '../src/trackers'
-
-const allowedMessages = [
-    'getClickToLoadState',
-    'getYouTubeVideoDetails',
-    'openShareFeedbackPage',
-    'setYoutubePreviewsEnabled',
-    'unblockClickToLoadContent',
-    'updateYouTubeCTLAddedFlag',
-    'updateFacebookCTLBreakageFlags'
-]
+import { AndroidMessagingConfig } from '../packages/messaging/index.js'
```

Comment:
should this be `import { AndroidMessagingConfig } from '@duckduckgo/messaging'`?

Reply:
Sure, done